lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -63,7 +63,7 @@ describe('workers/repository/process/lookup/index', () => {
     it('returns rollback for pinned version', async () => {
       config.currentValue = '0.9.99';
       config.depName = 'q';
-      config.datasource = datasourceNpmId;
+      config.datasource = NpmDatasource.id;
```

Comment:
We should use a default config, so we don't need to repeat / change this line in so many places. Can be done in a follow up PR.